oxd-server/src/main/resources/swagger.yaml

```diff
@@ -104,7 +104,7 @@ paths:
                   - refresh_token
                 properties:
                   scope:
-                    type: string
+                      type: string
```

Comment:
why do we need put extra space here? Spaces are super important in yaml, we should not put it without reason.